azure/apimanagement_service/dashboards/overview/main.tf

```diff
@@ -0,0 +1,226 @@
+terraform {
+  required_providers {
+    lightstep = {
+      source  = "lightstep/lightstep"
+      version = "~> 1.76.0"
+    }
+  }
+  required_version = ">= v1.0.11"
+}
+
+variable "lightstep_project" {
+  description = "Lightstep Project Name"
+  type        = string
+}
+
+output "dashboard_url" {
+  value       = "https://app.lightstep.com/${var.lightstep_project}/dashboard/${lightstep_dashboard.otel_collector_dashboard.id}"
+  description = "OpenTelemetry Collector Api Management Service Dashboard URL"
```

Comment:
do we usually say "Api" or "API" in our dashboards, which is the consistent way to refer to api?